.eslintrc.json

```diff
@@ -5,9 +5,7 @@
   "extends": [
     "plugin:@typescript-eslint/recommended",
     "plugin:monorepo/recommended",
```

Comment:
note: got rid of the .eslintrc.base concept for prettier and just made it always check prettier :) could revert but i think it's not too bad this way. i know i didn't like it at one point but seems ok now